cluster-sync/sync.sh

```diff
@@ -28,21 +28,27 @@ if [ "${KUBEVIRT_PROVIDER}" != "external" ]; then
   registry=${IMAGE_REGISTRY:-localhost:$(_port registry)}
   DOCKER_PREFIX=${registry}
   MANIFEST_REGISTRY="registry:5000"
+  QUAY_NAMESPACE="none"
 fi
 
 # Need to set the DOCKER_PREFIX appropriately in the call to `make docker push`, otherwise make will just pass in the default `kubevirt`
+QUAY_NAMESPACE=$QUAY_NAMESPACE DOCKER_PREFIX=$MANIFEST_REGISTRY PULL_POLICY=$(getTestPullPolicy) make manifests
 DOCKER_PREFIX=$DOCKER_PREFIX make docker push
-DOCKER_PREFIX=$MANIFEST_REGISTRY PULL_POLICY=$(getTestPullPolicy) make manifests
 
 seed_images
 
 configure_local_storage
 
 # Install CDI
-_kubectl apply -f "./_out/manifests/release/cdi-operator.yaml" 
+install_cdi
+
+#wait cdi crd is installed with 120s timeout
+wait_cdi_crd_installed 120
```

Comment:
Should probably make this a variable that can also be set via env variable.

Reply:
This is similar to timeout we have when waiting for CDI cr to be installed. This is utilized only during cluster-sync on kubevirtci, that is why I did not make variable out of it

Reply:
Well you just made it so this also works for when you cluster-sync to an external provider, and since we have little to no control over the external provider, it would be nice to give the external provider user as many knobs they can change to make it work for their cluster.

Reply:
> This is similar to timeout we have when waiting for CDI cr to be installed. This is utilized only during cluster-sync on kubevirtci, that is why I did not make variable out of it

Fair enough, not a big deal fro me, I was justthinking if it was a declared var at the top of the script file it's easy to identify and bump up or down if we need to in the future, or more importantly if somebody ever says "Hey Griffith, what's the timeout duration for cluster-sync?".  I can just open the file and "BOOM" it hits me right in the face.  Your call though I certainly wouldn't hold this up any longer over it.

Reply:
Added Variable for CD_INSTALL_TIMEOUT